docs/framework/wcf/feature-details/partial-trust-best-practices.md

```diff
@@ -10,12 +10,11 @@ This article describes best practices when running Windows Communication Foundat
 
 ## Serialization
 
-Apply the following practices when using the <xref:System.Runtime.Serialization.DataContractSerializer> in a partially trusted application.
+Apply these practices when using the <xref:System.Runtime.Serialization.DataContractSerializer> in a partially trusted application.
 
-- All serializable types must be explicitly marked with the `[DataContract]` attribute. The following techniques aren't supported in a partial trust environment:
+All serializable types must be explicitly marked with the `[DataContract]` attribute. The following techniques aren't supported in a partial trust environment:
 
 - Marking classes to be serialized with the <xref:System.SerializableAttribute>.
-
 - Implementing the <xref:System.Runtime.Serialization.ISerializable> interface to allow a class to control its serialization process.
```

Comment:
Is "partially trusted application" a reference to CAS, and if so should all this text just be deleted?

Reply:
The entire article is about partial trust - and we have a lot of docs/code related to partial trust still. I feel like that should be a separate issue/PR.